### PR TITLE
[7.3] PhpUnitExpectationFixer - support PHP 7.3

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Fixtures/Integration/misc/PHP7_3.test [hunk 1]
```diff
@@ -98,6 +102,7 @@ final class MyTest extends \PHPUnit_Framework_TestCase
 {
     public function testFoo(): void
     {
+        $this->setExpectedException('RuntimeException', 'Msg', 123); // `php_unit_expectation` rule
```
Comment (SpacePossum) [inline tests/Fixtures/Integration/misc/PHP7_3.test]: can you add a trailing ',' after `123` here?

Comment (kubawerlos) [reply tests/Fixtures/Integration/misc/PHP7_3.test]: Of course :man_facepalming: - done.

File: src/Fixer/PhpUnit/PhpUnitExpectationFixer.php [hunk 2]
```diff
@@ -161,7 +161,7 @@ protected function createConfigurationDefinition()
         ]);
     }
 
-    private function fixExpectation($tokens, $startIndex, $endIndex)
+    private function fixExpectation(Tokens $tokens, $startIndex, $endIndex)
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitExpectationFixer.php]: :+1:

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**